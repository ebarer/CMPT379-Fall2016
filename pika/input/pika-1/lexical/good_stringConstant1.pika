exec {
	const v := "now".
    const w := "is the".
    const x := "time for".
    const y := "all".
    const z := "good".
    print v; w; x; y; z, _n_.
}