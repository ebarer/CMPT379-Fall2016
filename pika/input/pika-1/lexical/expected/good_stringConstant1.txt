now is the time for all good
